airflow/www/api/experimental/endpoints.py

```diff
@@ -37,6 +37,9 @@
 
 api_experimental = Blueprint('api_experimental', __name__)
 
+# Possible yes values to replace_microseconds parameter for dag_runs (trigger_dag) endpoint
+YES_VALUES = ['true', '1', 't', 'y', 'yes']
```

Comment:
Re-order values index
```suggestion
YES_VALUES = ['true', 't', 'y', 'yes', '1']
```